appside/model/test_dbt_worksheet_model.js

```diff
@@ -145,3 +146,17 @@ function visual_test_self_report() {
     let view = new SelfReportFrame(frame);
     view.render();
 }
+
+/*
+ * Integration test that invokes ConsentDisclosureFrame to render the consent disclosure frame of this app.
+ * Manually verified.
+ */
+function visual_test_consent_disclosure() {
+    let model = new DbtWorksheetModelFwd(knowledgebase, FWD_PROMPTING_CONFIG);
```

Comment:
since consent will be disabled by default in the config, need to enable it on config before constructing model.

Reply:
current test is able to display visually the frame but the collected user input is never stored in model.